1161.py

```diff
@@ -0,0 +1,41 @@
+from collections import deque
+
+# Definition for a binary tree node.
+# class TreeNode:
+#     def __init__(self, val=0, left=None, right=None):
+#         self.val = val
+#         self.left = left
+#         self.right = right
+class Solution:
+    def maxLevelSum(self, root: TreeNode) -> int:
+        level_sums = []
+        queue = deque([root])
+        
+        while queue:
+            level_sum = 0
+            level_length = len(queue)
+            
+            while level_length > 0:
+                node = queue.popleft()
+                level_sum += node.val
+                
+                if node.left:
+                    queue.append(node.left)
+                
+                if node.right:
+                    queue.append(node.right)
+                
+                level_length -= 1
+
+            level_sums.append(level_sum)
+            level_sum = 0
+
+        min_level = 0
```

Comment:
You can maintain a pair of variables `max_sum` and `result_level`, update them in the `while` loop body (whenever you see a greater `level_sum`, you make this update), and delete all the code between line 33 and line 39.